src/Lfds/LfdsClient.cs

```diff
@@ -0,0 +1,447 @@
+//----------------------

```

Comment:
would be good to also commit the nswag.json file and swagger file used to auto generate this client so that we can regenerate it easily again in the future